### PR TITLE
add default arg example

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,10 +201,28 @@ <h3>Passing arguments defined on "this" component</h3>
...
+        </div>
+      </div>
+
+    </section>
+
     <section>
       <h3>All components are tagless</h3>
       <p>
-        In Octane, components don't have a default wrapper anymore, so you don't
+        In Octane, Glimmer components don't have a default wrapper, so you don't
```
Comment (rwjblue) [inline index.html:225]: I'm not sure this change is quite right (we mostly _do_ want folks to think of `@glimmer/component` as "all components"). At the least it doesn't match the H3 just above.

**Comment by locks** [comment]
Conflicts ahead!

File: index.html (hunk 1 above)
Comment (jenweber) [inline index.html:225]:
```suggestion
        In Octane, components don't have a default wrapper, so you don't
```

**Comment by jenweber** [comment]
Thank you @NullVoxPopuli! Merged after removing "Glimmer" per the comment above.

**Pull request merged**